project/plugins.sbt

```diff
@@ -8,3 +8,5 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 addSbtPlugin("com.sksamuel.scapegoat" %% "sbt-scapegoat" % "1.0.9")
 addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.0.3")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.18")
+addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.20")
+libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.9.0-RC1"
```

Comment:
Tbh we don't really depend on this version any more, we support all fresh versions now AFAIK